<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SelectParticipants</class>
    <widget class="QDialog" name="SelectParticipants">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>200</width>
                <height>400</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>200</width>
                <height>400</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>200</width>
                <height>400</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Select Participants</string>
        </property>
        <property name="windowIcon">
            <iconset resource="resources.qrc">
                <normaloff>:/resource/invite-large.png</normaloff>:/resource/invite-large.png
            </iconset>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="chatroomNameLabel">
                    <property name="font">
                        <font>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Chat room name:</string>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="chatroomName"/>
            </item>
            <item>
                <widget class="QLabel" name="label">
                    <property name="font">
                        <font>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Select user(s) to be added to chat room:</string>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QListWidget" name="allUserList">
                    <property name="selectionMode">
                        <enum>QAbstractItemView::ExtendedSelection</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="resources.qrc"/>
    </resources>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>SelectParticipants</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>190</x>
                    <y>390</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>SelectParticipants</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>190</x>
                    <y>390</y>
                </hint>
                <hint type="destinationlabel">
                    <x>199</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
